minorminer/utils/parallel_embeddings.py

```diff
@@ -101,17 +103,46 @@ def embeddings_to_array(embs: list, node_order=None, as_ndarray=False):
         return [[emb[v] for v in node_order] for emb in embs]
 
 
+def array_to_embeddings(embs: list, node_order=None):
```

Comment:
```suggestion
def array_to_embeddings(embs: list, node_order: Optional[Iterable] = None) -> list[dict]:
```
with
```python
from collections.abc import Iterable
```
added to imports.